assets/src/edit-story/app/highlights/quickActions/constants.js

```diff
@@ -52,10 +52,10 @@ export const ACTION_TEXT = {
 
 export const RESET_PROPERTIES = {
   ANIMATION: 'animation',
-  BACKGROUND_OVERLAY: 'backgroundOverlay',
```

Comment:
@BrittanyIRL FYI -- this PR is changing `backgroundOverlay` to `overlay` since overlay will be available for all media and not just the background. I'm changing the files here you just recently updated, I believe. Pinging just in case.

Reply:
thank you so much fore the heads up and for updating this!